autoencoder_model.py

```diff
@@ -0,0 +1,46 @@
+'''
+Description: This script provides the main autoencoder model structure that is used for GFS 2m temperature bias correction task.
```

Comment:
I am thinking this can be applied to other products too (e.g. precip). Can we change GFS 2m temperature to GFS forecast products?

Reply:
Updated!
Yes, it can be applied; however, we need to train it again.